### PR TITLE
Add prow configs for private-cluster-rhapi-apischeme-updater

### PR DESCRIPTION
**Pull request by csheremeta**

### CODE REVIEW

File: ci-operator/config/openshift/private-cluster-rhapi-apischeme-updater/openshift-private-cluster-rhapi-apischeme-updater-master.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: '4.0'
+    namespace: ocp
+    tag: base
```
Comment (cblecker) [inline ci-operator/config/openshift/private-cluster-rhapi-apischeme-updater/openshift-private-cluster-rhapi-apischeme-updater-master.yaml:6]:
```suggestion
    name: ubi
    namespace: ocp
    tag: "8"
```

File: ci-operator/config/openshift/private-cluster-rhapi-apischeme-updater/openshift-private-cluster-rhapi-apischeme-updater-master.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+images:
+- dockerfile_path: ./Dockerfile
```
Comment (cblecker) [inline ci-operator/config/openshift/private-cluster-rhapi-apischeme-updater/openshift-private-cluster-rhapi-apischeme-updater-master.yaml:8]:
```suggestion
- dockerfile_path: Dockerfile
```

File: ci-operator/config/openshift/private-cluster-rhapi-apischeme-updater/openshift-private-cluster-rhapi-apischeme-updater-master.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    tag: base
+images:
+- dockerfile_path: ./Dockerfile 
+  from: base
+  to: private-cluster-rhapi-apischeme-updater
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
```
Comment (cblecker) [inline ci-operator/config/openshift/private-cluster-rhapi-apischeme-updater/openshift-private-cluster-rhapi-apischeme-updater-master.yaml:15]:
```suggestion
    name: python
    namespace: openshift
    tag: "3.6"
```
```

**Comment by petr-muller** [comment]
/approve

Looks good from DPTP PoV, approving to allow merge when ready.

**Comment by cblecker** [comment]
@csheremeta Can you re-run `make jobs` on this? The generated config has updated.

**Comment by csheremeta** [comment]
functionality was implemented in https://github.com/openshift/release/pull/9651, closing this PR

**Pull request closed without merging**